.gitignore

```diff
@@ -157,3 +157,8 @@ tom_desc/admin_tools/elasticc2_dump_10obj.sql
 tom_desc/admin_tools/elasticc2_dump_100obj.sql
 tom_desc/admin_tools/elasticc2_dump_1000obj.sql
 tom_desc/elasticc_subset.sql
+
+# Locations of runtime files for a docker compose dev setup
+tom_desc/elasticc2/static/
```

Comment:
I'm a little bit nervous adding this to `.gitignore`.  Right now lots of things are generated during the run that get saved here, but it's also possible at some point that things that should be part of the archive will get put here.

(Probably what this means is that the things I save to static should be saved somewhere else.)

Reply:
FWIW you can `!tom_desc/elasticc2/static/filetokeep.txt` for those even without changing the directory structure or this line.

Ref: https://git-scm.com/docs/gitignore "Pattern Format" section.